api/proto/retriever/v2/retriever.proto

```diff
@@ -33,6 +34,7 @@ message BlobRequest {
 	uint32 quorum_id = 3;
```

Comment:
Does this param make any difference? Shouldn't it return the same blob no matter which quorum is specified for that blob?

Reply:
Yes. This determines which quorum the retriever will go to to fetch the blob. If the blob retrieval is successful, then different quorums would return the same data. But if the quorum specified here doesn't possess the blob, the request will not be successful.